mflowgen/Tile_MemCore/construct.py

```diff
@@ -315,6 +315,16 @@ def construct():
 
   if pwr_aware:
       pwr_aware_gls.update_params( { 'design_name': parameters['design_name'] }, True )
+     
+      init.extend_postconditions(         ["assert 'Clamping logic structure in the SBs and CBs is not maintained' in File( 'mflowgen-run.log' )"] )
```

Comment:
Does this pass? Shouldn't you be asserting that it _is_ maintained?

- assert 'your stuff is maintained' in File( 'mflowgen-run.log' )

or

- assert 'your stuff not maintained' not in File( 'mflowgen-run.log' )

Reply:
Yeah, I had pushed a change for it. Assertion is to check if the error message is _not_ in the file:

init.extend_postconditions(         ["assert 'Clamping logic structure in the SBs and CBs is not maintained' not in File( 'mflowgen-run.log' )"] )